libs/platform/README.md

```diff
@@ -82,18 +81,18 @@ For an existing Angular CLI application,
     import { ToggleModule } from '@fundamental-ngx/core';
     ```
 
+    Note: Be careful while importing the entire `FundamentalNgxPlatformModule` as it loads all modules; we recommend to only import relevant modules as needed.
+
     For models prior to 0.11.1 use `fundamental-ngx`
 
+    ```javascript
```

Comment:
does this make sense? Maybe you wanna give an example with a platform component instead.

Reply:
have added a platform component example instead.